client/js/jsonedit.js

```diff
@@ -651,11 +651,7 @@ async function jeApplyExternalChanges(state) {
 }
 
 async function jeClick(widget) {
-  if(jeState.ctrl) {
-    jeSelectWidget(widget);
-  } else {
-    await widget.click();
-  }
+  jeSelectWidget(widget);
```

Comment:
At this point, this function could just be removed.

Reply:
I did a search for jeState.ctrl and replaced with e.ctrlKey if in a keyboard event context (or in this case checked e.ctrlKey for the mouse event in mousehandling).

I wasn't sure if we'd want to call jeSelectWidget directly in mousehandling. With your new ctrl-shift-click we'll need additional changes. I'm thinking we might pass the mouse event to jeClick to be able to test additional modifier keys there in case we want to add meanings for ctrl-alt-click, etc.